Yoast/wpseo-woocommerce
# Retrieves noindex from shop page instead of product archive

## Summary

This PR can be summarized in the following changelog entry:

* Uses the filter `wpseo_sitemap_page_for_post_type_archive` to get noindex from shop page instead of product archive.

## Test instructions

This PR can be tested by following these steps:

* Install [WooCommerce](https://wordpress.org/plugins/woocommerce/), create and set "shop" page.
* Check both sitemaps (page and product). This page should be listed in both sitemaps.
* Set _noindex_ for "shop" page.
* Check both sitemaps (page and product). This page should be removed in both sitemaps.
